index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <h1>Book List</h1>
+
+    <form id="bookForm">
+      <label for="title">Title:</label>
+      <input type="text" id="title" required>
+  
+      <label for="author">Author:</label>
+      <input type="text" id="author" required>
+  
+      <button type="submit">Add Book</button>
+    </form>
+  
+    <h2>My Books</h2>
+    <hr>
+    <ul id="booksList"></ul>
+  
+    <script src="index.js"></script>
```

Comment:
You did well with implementing this milestone, please your project should follow the framework given, with the title and author below while the displayed items at the top just as it is in the framework 😄 

framework :
![image](https://github.com/Uzair-Manzoor/Awesome-books/assets/20537712/8c36dfb6-54f8-4fa0-a81b-05565a4e7bb0)

Your display :
![image](https://github.com/Uzair-Manzoor/Awesome-books/assets/20537712/56784f0a-4cf6-43f4-b587-caceb53068ca)